test/integration/scheduler/queue_test.go

```diff
@@ -713,14 +713,14 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			},
 			triggerFn: func(testCtx *testutils.TestContext) error {
 				// Trigger an Node update event.
-				// It should requeue pod4 only because this node only has zone label, and doesn't have node label that pod3 requires.
+				// It should requeue pod3, pod4, the update adds the "zone" label to fake-node2.
```

Comment:
```suggestion
				// It should requeue pod4 because the update adds the "zone" label to fake-node2.
				// Also, it should requeue pod3 because it deletes the "node" label from fake-node2.
```

Reply:
done